package.json

```diff
@@ -3,7 +3,7 @@
   "version": "1.0.0",
   "scripts": {
     "trunk": "trunk",
-    "test": "jest --runInBand"
+    "test": "jest"
```

Comment:
`npm test` does not work for me; `jest` is not installed into my `PATH` when I `npm install` locally.

```
sam@sam-dell:2022-12-19T18:20:22-0800:~/plugins2 (tyler/improve-testing-1) [1]
$ npm test
++ 2022-12-19T18:20:24-0800 ++ npm test

> trunkio-plugins@1.0.0 test
> jest

sh: 1: jest: not found
```

(maybe there's something wrong with my personal `PATH` setup and npm/PS1 hooks? not sure)

Reply:
Let's figure this out